specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -555,6 +555,183 @@
           }
         }
       }
+    },
+    "/weather/historical/actuals/daily/{format}": {
+      "get": {
+        "description": "**Get Daily Historical Actuals**\n  \n  \n**Applies to**: G1 and G2 pricing tiers.\n\n\nThe service returns climatology data such as past daily actuals for a given coordinate location.",
```

Comment:
To provide more details about what Actuals supports and how, I propose we change:

The service returns climatology data such as past daily actuals for a given coordinate location.

To:

Get Daily Historical Actuals service returns climatology data such as past daily actual observed temperatures, precipitation, snowfall, snow depth and cooling/heating degree day information, for the day at a given coordinate location. The data is requested for a specified date range, up to 31 days in a single API request. Generally, historical data may be available as far back as the last 5 to 40+ years, depending on the location.